LSTM_model.py

```diff
@@ -187,4 +224,12 @@ def model_settings(number_of_nodes, X_train, Y_train):
 
 
 # Real, = plt.plot(save_dates.values[:Y_test.shape[0]],Y_test)
-# Predict, = plt.plot(save_dates.values[:Y_test.shape[0]],predict)
\ No newline at end of file
+# Predict, = plt.plot(save_dates.values[:Y_test.shape[0]],predict)
+
+
+if __name__ == "__main__":
+	parser = argparse.ArgumentParser(description='LSTM supervised')
```

Comment:
It means that now you can run it with the following script params:

--timesteps_to_the_future
6